src/Sql/Platform/Mysql/Ddl/CreateTableDecorator.php

```diff
@@ -163,14 +163,13 @@ private function normalizeColumnOption($name)
     }
 
     /**
-     * @internal @private this method is only public for PHP 5.3 compatibility purposes.
      *
      * @param string $columnA
      * @param string $columnB
      *
      * @return int
      */
-    public function compareColumnOptions($columnA, $columnB)
+    private function compareColumnOptions($columnA, $columnB)
```

Comment:
The method had been marked as `@internal` and `@private` (I think by me). Can't have more warnings than that - `private` is perfectly fine here, and nobody should ever have relied on it in first place.